README.md

```diff
@@ -1,16 +1,21 @@
 # nestjs-json-rpc
 
-A JSONRPC microservice strategy implementation for NestJS.
+[Docs](https://github.com/hfour/nestjs-json-rpc/wiki/Documentation) |
+[Contributing](https://github.com/hfour/nestjs-json-rpc/wiki/Contributing) |
+[Wiki](https://github.com/hfour/nestjs-json-rpc/wiki) |
+[MIT Licensed](LICENSE.md)
+
+A JSON-RPC microservice strategy implementation for NestJS.
 
 Currently uses HTTP as the transport layer, with plans to add other options as the need arises.
 
 ## Install
 
-yarn add @hfour/nestjs-json-rpc
+`yarn add @hfour/nestjs-json-rpc`
 
 ## Usage example
 
-Initialize similar to a regular microservice, but pass a JSONRPCServier as the strategy option:
+Initialize similar to a regular microservice, but pass a `JSONRPCServier` as the strategy option:
```

Comment:
now that I look at it, it looks like I have a typo - it is not Servier

Reply:
Right!